lib/util/git/index.ts

```diff
@@ -753,24 +740,34 @@ export async function commitFiles({
           logger.warn({ err, fileName }, 'Cannot delete file');
           ignoredFiles.push(fileName);
         }
-      } else if (await isDirectory(join(localDir, file.name))) {
-        fileNames.push(file.name);
-        await gitAdd(file.name);
+      } else if (await isDirectory(join(localDir, fileName))) {
+        logger.warn({ fileName }, 'Skipping directory commit');
+        ignoredFiles.push(fileName);
```

Comment:
We previously used to add this but it does not seem right because our "source of truth" is our file list and not what's on disk.